nodemon.json

```diff
@@ -2,7 +2,7 @@
     "restartable": "rs",
     "verbose": false,
     "execMap": {
-      "ts": "node --require ts-node/register"
+      "ts": "node --require ts-node/register --require dotenv/config"
```

Comment:
Wow this reloads when env's are changed? Slick 👌

Reply:
Yeah I put there here so that env is loaded and configured before any of the app runs. Otherwise you have to do it in app.js